openeogeotrellis/load_stac.py

```diff
@@ -84,9 +84,28 @@ def supports_item_search(coll: pystac.Collection) -> bool:
         conforms_to = coll.get_root().extra_fields.get("conformsTo", [])
         return any(conformance_class.endswith("/item-search") for conformance_class in conforms_to)
 
+    def is_raster_mime_type(mime_type: str) -> bool:
```

Comment:
```suggestion
    def is_supported_raster_type(mime_type: str) -> bool:
```

Reply:
Renamed to `is_supported_raster_mime_type`